cfme/tests/test_appliance.py

```diff
@@ -373,6 +373,44 @@ def test_appliance_exec_scripts():
     pass
 
 
+@pytest.mark.manual
+@pytest.mark.tier(1)
+def test_puma_server():
+    """
+    From a bugPuma, by default is a threaded web server, which means each
```

Comment:
**Required** can you move these steps into `testSteps` and `expectedResults` in the polarion docblock?